isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -1838,6 +1846,13 @@ namespace Isis {
   }
 
 
+  Table Cube::readTable(const QString &name) {
+    Blob tableBlob(name, "Table");
+    read(tableBlob);
```

Comment:
Should there be a try catch around this read?

Reply:
Yeah I should add a try catch to rethrow the Blob error with a better message.